.github/workflows/test-kms.yml

```
@@ -0,0 +1,61 @@
+name: Run KMS tests
+
+on:
+  push:
+  workflow_dispatch:
+
+permissions: {}
+
+jobs:
+  test-kms:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: 'write' # for OIDC auth for GCP authentication
+      issues: 'write' # for filing an issue on failure
+
+    steps:
+      - name: Checkout securesystemslib
+        uses: actions/checkout@v2
```

Should we pin to commit as we do in ci.yml now?
```suggestion
        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
```